pkg/render/policyrecommendation.go

```diff
@@ -266,6 +315,15 @@ func allowTigeraPolicyForPolicyRecommendation(cfg *PolicyRecommendationConfigura
 			Destination: networkpolicy.ESGatewayEntityRule,
 		},
 	}
+
+	if !cfg.ManagedCluster {
```

Comment:
Is this check needed ? PolicyRec only runs inside a management and standalone cluster.

Reply:
The component only runs inside of management/standalone clusters, but the operator controller may run inside of a managed cluster for other reasons, I assume?